### ISSUE TITLE
[Suggestion] NBT Reader

### ISSUE THREAD

**Issue by Buuz135** [SonarSonic/Practical-Logistics#31]
The info reader doesn't provide all the info that sometimes you want to know. In GregTech case all the info is stored in NBT and the Info Reader doesn't get all the important info. The Gui can be the same as the Info one but with an extra field to name the NBT property so when it's displayed in the screens it has a proper name.

Also if you want, you can add Gregtech Integration :stuck_out_tongue:

**Comment by SonarSonic** [comment]
Yeah I had planned some sort of NBT reading. Hopefully this will be available sometime in the future.
